src/System.Security.Permissions/src/System/Xaml/Permissions/XamlLoadPermission.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Security.Permissions;
+
+namespace System.Xaml.Permissions
+{
+    [Serializable]
+    public sealed class XamlLoadPermission : CodeAccessPermission, IUnrestrictedPermission
+    {
+        private IList<XamlAccessLevel> _emptyAccessLevel = new ReadOnlyCollection<XamlAccessLevel>(Array.Empty<XamlAccessLevel>());
+        public XamlLoadPermission(PermissionState state) { }
+        public XamlLoadPermission(XamlAccessLevel allowedAccess) { }
+        public XamlLoadPermission(IEnumerable<XamlAccessLevel> allowedAccess) { }
+#if NETCOREAPP3_0
```

Comment:
Do we really need a `NETCOREAPP3_0` conditional? If so, consider using `NETCOREAPP` instead.

Reply:
There was a comment in `GetHashCode` that said the implementation was there to silence a compiler warning. Not sure why the `Equals` was also within the conditional, so I left it in. 
Updated conditional to NETCOREAPP.